### PR TITLE
Add OLS option for term_browser, add poro as test case

### LINKED ISSUE #534: Provide an easier way to use a custom browser for class PURLs
We have a growing number of opt-outs of ontobee in obo.yml. This is only likely to grow.

Ideally we would not have lots of people editing obo.yaml

It's not clear to me if it's actually necessary to edit obo.yml to redirect to OLS? I will try an experiment and link here.

### PR DESCRIPTION
**Pull request by cmungall**
Fixes #534

### CODE REVIEW

**Comment by jamesaoverton** [comment]
This looks good. Is it ready to merge?

**Comment by cmungall** [comment]
I didn't get round to testing this on my machine

File: config/poro.yml [hunk 1]
```diff
@@ -23,8 +23,8 @@ entries:
 - prefix: /about/
   replacement: http://www.ontobee.org/browser/rdf.php?o=PORO&iri=http://purl.obolibrary.org/obo/
```
Comment (jamesaoverton) [inline config/poro.yml]: This replacement still points to Ontobee, so the test that points to EBI fails (and with #544 we can now see the failure).

The `term_browser` field is specifically for the `http://purl.obolibrary.org/obo/ABC_123` case. There's no special handling of `/about/`.

**Comment by jamesaoverton** [comment]
Now I see that one of the PORO test is failing. The script changes adding `ols` ares fine. I want to add `ols` to `tools/test.py` as well.

**Comment by jamesaoverton** [comment]
@simonjupp For the term "PORO:0000017" the PURL is <http://purl.obolibrary.org/obo/PORO_0000017> and with this PR we'll redirect it to OLS at <https://www.ebi.ac.uk/ols/ontologies/poro/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPORO_0000017>. Is this right? In particular:

1. should we be lowercasing our `idspace: PORO` to `poro` in the middle of the URL?
2. should we be escaping slashes and stuff in the query part of the URL?

**Comment by matentzn** [comment]
@henrietteharmse @udp Can you guys help @jamesaoverton  out here? Thanks!

**Comment by udp** [comment]
Hello,

1. We use the IDs from https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/_config.yml which are indeed lowercase (poro)

2. Yes, these characters need to be escaped as they are part of the query string of the URL.

Let me know if you have any more questions.

**Comment by jamesaoverton** [comment]
Thanks @udp and @matentzn!

**Pull request merged**